### DISCUSSION TITLE
Missing dist directory in v3.

### DISCUSSION THREAD

**Discussion by Amplificator** [tailwindlabs/tailwindcss#7029] [Help] [answered]
Hi, where did the dist directory go in v3?

I was using the full .css files for development purposes before building for production and keeping only the needed classes.

Is this intentional? I can't seem to find any information regarding this?

**Comment by Amplificator** [comment]
In the meanwhile I am using the Play CDN for development but I'm still interested in knowing why the dist directory is gone :)

**Comment by adamwathan** [comment, marked as answer]
See prior discussion here 👍

https://github.com/tailwindlabs/tailwindcss/discussions/6256

**Reply by Amplificator** [reply]
Alright, I understand now.. Thank you :)
